packages/webview_flutter/webview_flutter_platform_interface/test/webview_flutter_platform_interface.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart'
+    as main_file;
+
+void main() {
+  group('webview_flutter_platform_interface', () {
+    test(
+      'ensures webview_flutter_platform_interface.dart exports classes in types directory',
+      () {
+        // ignore: unnecessary_statements
```

Comment:
You could make this test its own file (e.g., `webview_flutter_platform_interface_export_tests.dart`), and then make this a file-level ignore, since a file that only tests exports wouldn't need to worry about losing lint coverage.

Reply:
Good suggestion. I will also make a mental note to do this for the `webview_flutter` package as well.